.env.local.example

```diff
@@ -36,3 +36,5 @@ ENABLE_MAILER=false
 MAILER_DOMAIN=
 MAILER_API_KEY=
 ROOM_INV_EMAIL_TEMPLATE=
+
+ROOM_PERSISTANT_VOLUME_PATH=./volume
```

Comment:
Don't forget to add other envs in this example. For example, `PORT` and `LOCAL_STORAGE_PATH` haven't been specified here